src/main/java/com/blackduck/integration/detect/lifecycle/boot/DetectBootFactory.java

```diff
@@ -242,5 +242,9 @@ public InteractiveManager createInteractiveManager(List<PropertySource> property
         InteractiveModeDecisionTree interactiveModeDecisionTree = new InteractiveModeDecisionTree(detectInfo, blackDuckConnectivityChecker, propertySources, gson);
         return new InteractiveManager(propertySourceBuilder, writer, interactiveModeDecisionTree);
     }
+    
+    public void stripCorrelationiD() {
```

Comment:
I'd fix casing in this and another similar method but overall changes look good.

Reply:
Done, thanks!